joern-cli/frontends/javasrc2cpg/src/test/scala/io/joern/javasrc2cpg/querying/TypeTests.scala

```diff
@@ -129,6 +129,28 @@ class NewTypeTests extends JavaSrcCode2CpgFixture {
         case result => fail(s"Expected identifier and arrayInitializer arguments but got $result")
       }
     }
+
+    // This test relies on instability in JavaParser results, so running it multiple times is a way of increasing
+    // confidence in the correctness of the code, albeit at the cost of slower tests.
```

Comment:
That is not correct anymore. So running the test once is enough. I would also expect a different lambda signature in the test below.